test/unit/spec/v2/ion/uiSchemaTransformer_spec.js

```diff
@@ -428,32 +387,17 @@ describe('v2/ion/uiSchemaTransformer', function () {
             ],
             'uiSchema': [
               {
-
                 'name': 'authenticator',
                 'type': 'authenticatorVerifySelect',
                 'label-top': true,
                 'options': [
                   {
-                    'label': 'Okta Password',
+                    'label': 'Primary Email Address',
                     'value': {
-                      'id': 'aidwboITrg4b4yAYd0g3'
+                      'id': 'aut1bospdDFs7q3vc0g4'
                     },
-                    'authenticatorType': 'password'
+                    'authenticatorType': undefined,
```

Comment:
@haishengwu-okta : Is this ok? Or can we open a separate ticket to fix this?

Reply:
it shall not be undefined. it's probably the issue that I'm trying to fix at https://github.com/okta/okta-signin-widget/pull/1254
Any chance you can avoid to change mocks and test?

Reply:
Yes, it indeed was, rebased it after your merge and fixed it..